### PR TITLE
fix: UX should not eagerly store strategy updates!

### PR DESCRIPTION
**Pull request by ivarconr**

### CODE REVIEW

File: src/component/feature/strategy/strategies-list-component.jsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+import arrayMove from 'array-move';
+import { Button, Icon } from 'react-mdl';
+
+import ConfigureStrategy from './strategy-configure-container';
+import AddStrategy from './strategies-add';
+import { HeaderTitle } from '../../common';
+import { updateIndexInArray } from '../../common/util';
+import styles from './strategy.module.scss';
+
+const randomKeys = length => Array.from({ length }, () => Math.random());
```
Comment (ivarconr) [inline src/component/feature/strategy/strategies-list-component.jsx:16]: not needed anymore as we are not allowing re-order while updating activation strategies. I will clean this up.

File: src/component/feature/strategy/strategy-configure-component.jsx [hunk 2]
```diff
@@ -164,17 +143,23 @@ export default class StrategyConfigureComponent extends React.Component {
                             </Link>
                             {editable && (
                                 <IconButton
-                                    title="Remove strategy from toggle"
+                                    title="Remove this acivation strategy"
```
Comment (chriswk) [inline src/component/feature/strategy/strategy-configure-component.jsx:146]:
```suggestion
                                    title="Remove this activation strategy"
```

File: src/component/feature/strategy/strategies-list-component.jsx (hunk 1 above)
Comment (ivarconr) [reply src/component/feature/strategy/strategies-list-component.jsx]: fixed

**Review by chriswk — APPROVED** [review]
I like it, should avoid prematurely enabling toggles.

File: src/component/feature/strategy/strategies-list-component.jsx (hunk 1 above)
Comment (chriswk) [reply src/component/feature/strategy/strategies-list-component.jsx]: :+1:

**Pull request merged**